content/docs/deployment/csmoperator/release/_index.md

```diff
@@ -31,3 +31,4 @@ Description: >
 | Issue | Workaround |
 |-------|------------|
 | When CSM Operator creates a deployment that includes secrets (e.g., application-mobility, observability, cert-manager, velero), these secrets are not deleted on uninstall and will be left behind. For example, the `karavi-topology-tls`, `otel-collector-tls`, and `cert-manager-webhook-ca` secrets will not be deleted. | This should not cause any issues on the system, but all secrets present on the cluster can be found with `kubectl get secrets -A`, and any unwanted secrets can be deleted with `kubectl delete secret -n <secret-namespace> <secret-name>`|
+| In certain environments, users have encountered difficulties in installing drivers using the CSM Operator due to the 'OOM Killed' issue. This issue is attributed to the default resource requests and limits configured in the CSM Operator, which fail to meet the resource requirements of the user environments. | Before deploying the CSM Operator, it is crucial to adjust the memory and CPU requests and limits in the files https://github.com/dell/csm-operator/blob/main/config/manager/manager.yaml#L100, https://github.com/dell/csm-operator/blob/main/deploy/operator.yaml#L1330 to align with the user's environment requirements. If the containers running on the pod exceed the specified CPU and memory limits, the pod may get evicted.|
```

Comment:
It is unclear how to install the Operator with these settings.
Is it something we can tune in Openshift once the Operator is installed ? Or do we have to recompile the Operator ?

Reply:
Hi @coulof Thanks for the review. I have updated the documentation. Can you please recheck? Thanks